docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.7"
+services:
+  php-fpm:
+    volumes:
+      - spddata:/var/www/html/ 
+    build: 
+      context: . 
+      dockerfile: 'docker/php-fpm/Dockerfile'
+    networks:
+      - spdashboard
+    container_name: fpm
+  web:
+    build:
+      context: 'docker/httpd/'
+    depends_on:
+      - php-fpm
+    expose:
+      - "80"
```

Comment:
Consider to use https so when developing https quirks could also be encountered.

Reply:
That's certainly a good idea, but involves some hassles in creating and maintaining certificates. I've added it as a chore on the backlog